### ISSUE TITLE
pf-list-view custom-scope attributute no longer passing objects

### ISSUE THREAD

**Issue by AllenBW** [patternfly/angular-patternfly#478]
So [here's the implementation](https://github.com/manageiq/manageiq-ui-service/blob/773c7c0738eaaa96b6308e8ee0dc595fb14d81a1/client/app/services/service-details/service-details.html#L172)

And [here's the pr where we're making the pf4 leap](https://github.com/ManageIQ/manageiq-ui-service/pull/804)

`custom-scope` previously passed the contents of `vm` for ang-pf 3.x but now it doesn't... [looking at the file](https://github.com/patternfly/angular-patternfly/blob/master/src/views/listview/list-view.component.js#L455), I can't for the life of me see anything terribly offensive...

TL;DR  3.x -> 4.x change from attribute to element, all things should work, did that, something doesn't work, thought it might be worth a collective 👀

**Comment by AllenBW** [comment]
even passing a simple test object into `custom-scope` does not yield results on `customScope` in the transcluded content
